### DISCUSSION TITLE
pokedex

### DISCUSSION THREAD

**Discussion by chistoperez** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#427] [Show and tell]
Cómo ven mi pokedex responsive?
https://chistoperez.github.io/pokedex/

<img src="https://user-images.githubusercontent.com/99354481/158707832-905938a9-907c-4b25-9894-cbe3fc6a31f0.png" alt="drawing" width="700"/><img src="https://user-images.githubusercontent.com/99354481/158707909-5ab50578-d8f0-4ad5-8a95-11c73c5b4be6.png" alt="drawing" width="700"/>

<img src="https://user-images.githubusercontent.com/99354481/158709747-df3248df-21f0-444c-a2d7-f1e1a36a800f.png" alt="drawing" width="333"/><img src="https://user-images.githubusercontent.com/99354481/158709865-f5e505dd-d844-40cd-b569-a348f0005ab1.png" alt="drawing" width="333"/><img src="https://user-images.githubusercontent.com/99354481/158709998-cda52511-2f47-49eb-9b8c-a37c5fbba50f.png" alt="drawing" width="333"/>

**Comment by KarenOrozco** [comment]
está genial
![charmander-pokedex](https://user-images.githubusercontent.com/33531567/158712316-a7d59454-218c-4748-a3b2-9d112152f754.JPG)

**Comment by ValesApps** [comment]
esta geniaaaaal
